src/wovnio/wovnphp/Logger.php

```diff
@@ -37,7 +37,7 @@ public static function set($logger)
     public function __construct($token, $quiet = true, $prefix = 'WOVN.php')
     {
         $this->uniqueId = $token . $this->getReasonableCurrentTime();
-        date_default_timezone_set('UTC');
+        @date_default_timezone_set(@date_default_timezone_get());
```

Comment:
date_default_timezone_get() will return any user-set timezones, and if none is available it will fallback to UTC (but will also emit a warning).

details: https://www.php.net/manual/en/function.date-default-timezone-get.php